board-games/board-games/Model/CommonEntities/Achievement.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace board_games.Model.CommonEntities
+{
+    internal enum GameCategory
+    {
+        SkillIssueBro,
+        GameOfLife
+    }
+
+    internal class Achievement
+    {
+        private int _achievementId;
+        private string _name;
+        private string _description;
+        private GameCategory _gameCategory;
+
+        public Achievement(int achievementId, string name, string description, GameCategory gameCategory)
+        {
+            _achievementId = achievementId;
+            _name = name;
+            _description = description;
+            _gameCategory = gameCategory;
+            
+        }   
+
+        public bool CheckAchievementCompletition(int playerId) {
+            // database call
+            
+            return false;
+        }
+
+        public string GetNameOfAchievement()
```

Comment:
I think "GetName" is enough, because when we call achievement.GetName() we know that it's an achievement. Same for other similar methods